compiler/arm/env/OMRCPU.hpp

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef OMR_ARM_CPU_INCL
+#define OMR_ARM_CPU_INCL
+
+/*
+ * The following #define and typedef must appear before any #includes in this file
+ */
+#ifndef OMR_CPU_CONNECTOR
+#define OMR_CPU_CONNECTOR
+namespace OMR { namespace ARM { class CPU; } }
+namespace OMR { typedef OMR::ARM::CPU CPUConnector; }
+#else
+#error OMR::ARM::CPU expected to be a primary connector, but an OMR connector is already defined
+#endif
+
+#include "compiler/env/OMRCPU.hpp"
+
+
+namespace OMR
+{
+
+namespace ARM
+{
+
+class CPU : public OMR::CPU
+   {
+protected:
```

Comment:
Indentation seems off again:

```suggestion
   protected:
```

Reply:
Ditto, as above.